tests/EdFi.Common.UnitTests/Security/SecurePackedHashProviderTests.cs

```diff
@@ -40,6 +41,7 @@ protected override void Arrange()
 
                 var secureHasher = A.Fake<ISecureHasher>();
 
+                A.CallTo(() => secureHasher.GetAlgorithmHashCode).Returns(123);
```

Comment:
Similar here, need a test to verify its picking the right secure hasher, unless we go to a provider pattern.

Reply:
i added the provider instead